### PR TITLE
fix/footer

### PR DESCRIPTION
**Pull request by ponrajaprabhusk**
### Functionality:
fixed the footer in homepage

### Solution:
made changes in Bootstrap to make it responsive and fixed the scroller

### Risk level:
- [ ] high 
- [ ] medium
- [X] low

### How to test:
check the footer scroller and responsiveness

### CODE REVIEW

**Comment by Pendyala-thanvi** [comment]
@ponrajaprabhusk fill the **solution** and **how to fix** section

File: functions/model/contributors/lib.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ exports.getContributors = function() {
  * @return {any}
  */
 exports.setContributors = function(docId, email, about, photoUrl, title, name) {
+    console.log(about);
```
Comment (printf-twinkle) [inline functions/model/contributors/lib.js:53]: remove this console log

File: functions/model/contributors/tark/addContributor.js [hunk 2]
```diff
@@ -30,6 +30,8 @@ exports.addContributor = function(request, response) {
     const title = request.body.data.title;
     const name = request.body.data.name;
 
+    console.log(about);
```
Comment (printf-twinkle) [inline functions/model/contributors/tark/addContributor.js:33]: remove this console log

**Review by printf-twinkle — COMMENTED** [review]
Fixes

File: functions/model/contributors/lib.js (hunk 1 above)
Comment (ponrajaprabhusk) [reply functions/model/contributors/lib.js]: Done!

File: functions/model/contributors/tark/addContributor.js (hunk 2 above)
Comment (ponrajaprabhusk) [reply functions/model/contributors/tark/addContributor.js]: Done

**Comment by ponrajaprabhusk** [comment]
> 

D

File: functions/package-lock.json [hunk 3]
```diff
@@ -8,8 +8,8 @@
       "dependencies": {
         "cors": "^2.8.5",
         "fastify": "^3.25.0",
-        "firebase-admin": "^10.0.2",
```
Comment (viv9k) [inline functions/package-lock.json:11]: Revert the changes in this file

File: functions/package.json [hunk 4]
```diff
@@ -16,8 +16,8 @@
   "dependencies": {
     "cors": "^2.8.5",
     "fastify": "^3.25.0",
-    "firebase-admin": "^10.0.2",
-    "firebase-functions": "^3.20.1"
+    "firebase-admin": "^9.2.0",
+    "firebase-functions": "^3.19.0"
```
Comment (viv9k) [inline functions/package.json:20]: revert this file as well

**Review by viv9k — CHANGES_REQUESTED** [review]
Please revert the unnecessary changes

File: README.md [hunk 5]
```diff
@@ -11,20 +11,18 @@ _At **Worktez**, we are envisioned to give milestones to your dreams._
...
+With an agile platform like WorkTez, you can be affirmed by a more organised remote team management.You can assign, prioritise and fix feasible deadlines so that completion of tasks is not an ordeal anymore! 
 
 
-Worktez is a Web App developed by a team of fresh and enthusiastic minds to help plan, track and automate the development activities.
-Our focus is create visually appealing and functionally progressive solution to all your Project Management related work.
-WorkTez promotes a happy work culture environment because a happy employee is way more productive in comparison to an employee burdened with loads of work!
-Our solution is focused on agile based devOps automation, that helps in more organised remote team management. You can assign, prioritise and fix feasible deadlines so that completion of tasks is not an ordeal anymore!
-
 
 # Technologies used
```
[the remark this answers is not in the document]
Comment (ponrajaprabhusk) [reply README.md:21]: Done!

File: functions/package-lock.json (hunk 3 above)
Comment (ponrajaprabhusk) [reply functions/package-lock.json:11]: Done

File: functions/package.json (hunk 4 above)
Comment (ponrajaprabhusk) [reply functions/package.json:20]: Done

**Pull request closed without merging**